src/libraries/System.Private.Uri/src/System/Uri.cs

```diff
@@ -114,10 +116,22 @@ private enum Flags : ulong
             FragmentIriCanonical = 0x40000000000,
             IriCanonical = 0x78000000000,
             UnixPath = 0x100000000000,
+
+            CustomParser_ParseMinimalAlreadyCalled = 1UL << 62,
+
+            Debug_LeftConstructor = 1UL << 63
```

Comment:
We should use the same style value for these as the rest of the enum.  If you want to use the shift notation in general, let's update the rest accordingly.